controllers/fenceagentsremediation_controller.go

```diff
@@ -325,6 +335,41 @@ func (r *FenceAgentsRemediationReconciler) updateStatus(ctx context.Context, far
 	return nil
 }
 
+func (r *FenceAgentsRemediationReconciler) collectSecretParams(far *v1alpha1.FenceAgentsRemediation, ctx context.Context) (map[string]string, error) {
+	secretParams := map[string]string{}
+	nodeName := getNodeName(far)
+	nodeSecretName := fmt.Sprintf("fence-agents-credentials-node-%s", nodeName)
```

Comment:
This string is not referenced *anywhere*, meaning no-one can use this feature.

Please make the user specify this name, rather than to have them scour yet-to-be created documentation to find out about it.

Reply:
Yep here it is

>FAR CRD includes a new spec field, CredentialsSecretName, for the secret name, which is impractical when there are many secrets.

I was actually considering taking a hybrid approach  adding the Shared Secret to the spec and just the Node Name Secret prefix to the Spec (suffix is the node name).

Updated the design [here](https://docs.google.com/document/d/1O6EBqvTivEc9YjhFk5gB9a6BamRyMPe2NYAVluooZjk/edit?tab=t.0#heading=h.1p69gnjejvtd)  as a 3rd option.

Reply:
I've taken a crack at the first approach.
If you decide you have a different strong preference  let me know.

Reply:
I was thinking of the admin specifying the full secret name, not just a prefix

Reply:
This gives them the ability to put everything in one secret, have one secret per field, per node, whatever suits their style

Reply:
I've updated my suggestion in the [design](https://docs.google.com/document/d/1O6EBqvTivEc9YjhFk5gB9a6BamRyMPe2NYAVluooZjk/edit?tab=t.0#heading=h.1p69gnjejvtd) at the 4th option.

@slintes let me know if that makes sense.

Reply:
Yeah he was trying to capture maximum user flexibility option, where the trade-off is simplicity.
I've talked with him and he is fine with proceeding with the current design (my apologizes for forgetting to update).